array-methods/filter.ts

```diff
@@ -0,0 +1,22 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna',
+];
+
+const evenNumbers = numbers.filter((element) => element % 2 === 0);
+console.log('filter() evenNumbers: ', evenNumbers);
+
+const namesNoD: string[] = [];
+const nonDNames = names.filter((element) =>
+  element.includes('d') || element.includes('D') ? '' : namesNoD.push(element)
```

Comment:
You should not be creating a new array `namesNoD`. `filter` already creates a new array for you, so you should be utilizing the array that `filter` returns